packages/core/types/Visualization.ts

```diff
@@ -0,0 +1,21 @@
+export type Visualization = {
+  visualizationType: string
+  dataKey: string
+  formattedData: any
+  dataDescription: string
```

Comment:
I made changes to the Visualization Type in dashboard before. https://github.com/CDCgov/cdc-open-viz/pull/891/files#diff-895581a3bc8b13625dd553922f0d9eb5ca168f28f2c034835638fb96bfebd745

Reply:
I discovered dataDescripton is an Object

Reply:
I copied in the new visualization type and will rebase after your changes reach test.